ad-hoc/review_active_cases.R

```diff
@@ -0,0 +1,37 @@
+rm(list=ls())
+library(tidyverse)
+library(behindbarstools)
+library(lubridate)
+
+all_data <- read_scrape_data(all_dates = T)
+
+all_active <- all_data %>%
+    # look at all jurisdictions
+    # only look at these three variables
+    select(Facility.ID, Date, Residents.Active) %>%
+    # remove anything with missing data
+    na.omit() %>%
+    # for each date floor to the nearest month
+    mutate(Month = floor_date(Date, unit = "month")) %>%
+    # group by each facility and each month
+    group_by(Facility.ID, Month) %>%
+    # only get the first monthly observation for each facility
+    filter(1:n() == 1) %>%
+    # only get data that fall within these date ranges
+    filter(Month >= ymd("2021-04-01")) %>%
+    filter(Month <= ymd("2022-01-01")) %>%
```

Comment:
Can this date be a variable to make this more reusable? Maybe the variable defaults to system date?